CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 - Corrected grammer issues and clarified comments in all existing Ruby-based Windows Check and Metric files.
 - Moved existing Ruby-based Windows Check and Metric files to bin/ruby.
 - Changed file naming to the singular instead of the difference in plurality between 'check' and 'metrics'.
+- Dependency on Powersehell version 3.0 or above for powershell checks.
```

Comment:
This should be added under `## [Unreleased]`, with this being a breaking change so I would also add `### Breaking Change`. I did not realize we were 0.x.y of this gem released so I would like to release a 1.0 so that we can rev this to 2.0 and not break anyone who are pessimistically pinning minor version of gem which is sane and recommended. I will reply to this comment once I have released a 1.0 so you can rebase again (sorry for the confusion).